config/config.go

```diff
@@ -896,6 +907,7 @@ type Receiver struct {
 	SNSConfigs       []*SNSConfig       `yaml:"sns_configs,omitempty" json:"sns_configs,omitempty"`
 	TelegramConfigs  []*TelegramConfig  `yaml:"telegram_configs,omitempty" json:"telegram_configs,omitempty"`
 	WebexConfigs     []*WebexConfig     `yaml:"webex_configs,omitempty" json:"webex_configs,omitempty"`
+	MSTeamsConfigs   []*MSTeamsConfig   `yaml:"msteams_configs,omitempty" json:"teams_configs,omitempty"`
```

Comment:
Should we align the configuration name? I am wondering why we call `msteams` for yaml but `teams` for json.
@zhan9san @simonpasquier

Reply:
@yeya24 

Thanks for pointing it out. It has to be a typo.

I'll submit a PR to fix it.